pedal/gpt/setup.py

```diff
@@ -0,0 +1,25 @@
+"""
+Initialize and setup the OpenAI tool.
+"""
+
+from pedal.core.report import MAIN_REPORT, Report
+from pedal.gpt.constants import TOOL_NAME
+
+#we need setup.py?
+def reset(report=MAIN_REPORT):
+    """
+    Remove all settings for the OpenAI tool.
+
+    Args:
+        report: The report object
+
+    Returns:
+        This tool's data
+    """
+    report[TOOL_NAME] = {
+        'feedback': []
```

Comment:
The prompt is kind of hardcoded right now so it can choose whether or not to add feedback to the report, we could have a lambda function per prompt to process the results if it's successful